missions/W1/hello_world.py

```
@@ -0,0 +1,17 @@
+import functools
```

Ruff (0.8.2)

`functools` imported but unused

Remove unused import: `functools`

(F401)